lib/xdrgen/ast/declarations/string.rb

```diff
@@ -3,9 +3,11 @@ class String < Base
     delegate :name, to: :identifier
 
     def size
-      size_str = size_spec.size_t.text_value
+      size_spec.size
+    end
 
-      size_str.to_i if size_str.present?
+    def fixed?
+      size_spec.is_a?(Xdrgen::AST::VarSize)
```

Comment:
A string is _never_ fixed size in XDR... all strings are variable sized according to the spec.